wasm-rpc-stubgen/src/lib.rs

```diff
@@ -40,6 +41,12 @@ pub enum Command {
     /// Initializes a Golem-specific cargo-make configuration in a Cargo workspace for automatically
     /// generating stubs and composing results.
     InitializeWorkspace(InitializeWorkspaceArgs),
+    /// TODO
+    #[cfg(feature = "unstable-dec-dep")]
+    Declarative {
```

Comment:
I hope this won't be the final name of the command :)

Reply:
no, this is just the WIP name while it is still behind a feature